articles/azure-functions/durable-functions-external-events.md

```diff
@@ -41,7 +41,7 @@ public static async Task Run(
 }
 ```
 
-The preceding example listens for a single event and takes action when it's received.
+The preceding example listens for a specific single event and takes action when it's received.
```

Comment:
It doesn't listen and take action on any single event, it only takes action on a specific event